asgiref/timeout.py

```diff
@@ -115,14 +113,15 @@ def _cancel_task(self) -> None:
             self._cancelled = True
 
 
-def current_task(loop: asyncio.AbstractEventLoop) -> "asyncio.Task[Any]":
-    if PY_37:
-        task = asyncio.current_task(loop=loop)  # type: ignore
+def current_task(loop: asyncio.AbstractEventLoop) -> "Optional[asyncio.Task[Any]]":
```

Comment:
it might be worth updating the vendoring here: https://github.com/aio-libs/async-timeout/blob/master/async_timeout/__init__.py#L205-L209